src/browser/extension/options/options.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import FilterGroup from './FilterGroup';
+import AllowToRunGroup from './AllowToRunGroup';
+import MiscellaneousGroup from './MiscellaneousGroup';
+
+export default (props) => (
+  <div>
+    <FilterGroup {...props} />
```

Comment:
Split the `Options` component into three subcompontents and moved them to separate files because the component got too long.